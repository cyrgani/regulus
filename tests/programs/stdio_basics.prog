_(
    print("hello world 1!"),
    debug(catch(error("mistake"))),
    print("hello world 2!"),
)