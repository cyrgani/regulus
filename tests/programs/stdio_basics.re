print("hello world 1!"),
print("hello world 2!"),
write("foo"),
write(" "),
write(""),
write("bar"),
write(list(1, 2)),