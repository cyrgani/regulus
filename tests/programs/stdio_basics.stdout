hello world 1! 
String("fooError: mistake\nat programs/stdio_basics.re:2:33\nat programs/stdio_basics.re:2:27\nat programs/stdio_basics.re:2:21")
hello world 2! 
foo bar[1, 2]